app/routes/cdo/edit/change-status.js

```diff
@@ -73,5 +96,42 @@ module.exports = [
         }
       }
     }
+  },
+  {
+    method: 'GET',
+    path: `${routes.inBreach.get}/{indexNumber}/{dummy?}`,
+    options: {
+      auth: { scope: anyLoggedInUser },
+      handler: async (request, h) => {
+        const cdo = await getCdo(request.params.indexNumber)
+
+        if (cdo == null) {
+          return h.response().code(404).takeover()
+        }
+        const backNav = addBackNavigation(request, false, true)
+
+        const breachCategories = await getBreachCategories()
+
+        return h.view(views.inBreachCategories, new InBreachViewModel(cdo.dog, breachCategories, [], backNav))
+      }
+    }
+  },
+  {
+    method: 'POST',
+    path: `${routes.inBreach.post}/{indexNumber}`,
+    options: {
+      validate: {
+        payload: validateBreachReasonPayload,
+        failAction: breachReasonPostFailAction
+      },
+      auth: { scope: anyLoggedInUser },
+      handler: async (request, h) => {
+        const payload = request.payload
+
+        await setDogBreaches(payload, getUser(request))
+
+        return h.redirect(`${routes.changeStatusConfirmation.get}/${payload.indexNumber}`)
```

Comment:
You might want to keep the srcHashParam for the link to view dog record (so it keeps the proper history) but also not important if it is lost here